src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public class Ball {
+    public String colour;
+    public int number;
+
+    public void toString();

```

Comment:
Method `toString()` should contains the Ball info. Try to create the autogenerated one by Interllij idea.